### ISSUE TITLE
How to evaluate performances in two-stage-decision

### ISSUE THREAD

**Issue by NicoBiagi** [expfactory/expfactory#174]
## Version of Experiment Factory:

Latest version, pulled from GitHub

### Expected behavior

At the end of the experiment, the csv with the behavioural data gets downloaded, but I don't know which columns I need to look at in order to evaluate the performance

### Actual behavior

Columns 'credit_var' and 'performance_var' in the downloadedare empty

### Steps to reproduce behavior

Write here.

**Comment by vsoch** [comment]
This sounds like something to figure out for the experiment, not necessarily the deployment mechanism?

**Comment by NicoBiagi** [comment]
I agree. Do you know where I can find info on how to analyse the data for this specific task?

**Comment by vsoch** [comment]
What is the experiment?

**Comment by NicoBiagi** [comment]
The 'two-stage-decision' (https://expfactory.github.io/experiments/e/two-stage-decision/). I downloaded the folder and hosted it on a private server using CPanel.

**Comment by vsoch** [comment]
Your best bet is to ask at https://github.com/expfactory/expfactory-experiments and you can see the folks that have worked on that experiment over the years: https://github.com/expfactory/expfactory-experiments/commits/master/two_stage_decision. I didn't make most of the experiments, just packaged them up, so I'm afraid I don't know the answer to questions about them.
